dev/create-release/release-util.sh

```diff
@@ -73,7 +73,7 @@ function fcreate_secure {
 }
 
 function check_for_tag {
-  curl -s --head --fail "$ASF_REPO_WEBUI;a=commit;h=$1" >/dev/null
+  ! curl -s --fail "$ASF_REPO_WEBUI;a=commit;h=$1" | grep '404 Not Found' > /dev/null
```

Comment:
What does the `!` do -- make this fail if grep successfully found a 404? does `grep -v` get the same effect? No need to change it, it's fine, just curious

Reply:
Thank you for review and approval, @srowen . Yes, right. `!` does.
We cannot use `grep -v` which matches all the other lines doesn't have `404 Not Found`. `grep` only returns `1` when no lines are selected.

Reply:
Oh right, duh, wasn't thinking it through